src/GuiBase/BaseApplication.cpp

```diff
@@ -122,9 +128,10 @@ BaseApplication::BaseApplication( int argc, char** argv, const WindowFactory& fa
 
     std::time_t startTime = std::time( nullptr );
     std::tm* startTm = std::localtime( &startTime );
-    Ra::Core::StringUtils::stringPrintf( m_exportFoldername, "%4u%02u%02u-%02u%02u",
-                                         1900 + startTm->tm_year, startTm->tm_mon + 1,
-                                         startTm->tm_mday, startTm->tm_hour, startTm->tm_min );
+
+    std::stringstream ssTp;
+    ssTp << std::put_time(startTm, "%Y%m%d-%T");
+    QDir().mkdir( ssTp.str().c_str() );
```

Comment:
As we are now using filesystem, perhaps it should be better to use ``std::filesystem::create_directory(ssTp.str()); `` instead of ``QDir... ``

Reply:
You are right, I will.